docker/cloudsim_bridge/run_bridge.bash

```diff
@@ -21,4 +21,4 @@ export ROSCONSOLE_STDOUT_LINE_BUFFERED=1
 # atop process monitoring
 atop -R -w /home/developer/.ros/atop_log &
 
-ign launch cloudsim_bridge.ign -v 4 "$@" 2>&1 | tee /home/developer/.ros/bridge.log
+unbuffer -p ign launch cloudsim_bridge.ign -v 4 "$@" 2>&1 | tee /home/developer/.ros/bridge.log
```

Comment:
I think you don't need the `-p` if the `unbuffer` command is at the front, i.e. not after the pipe

Reply:
Thanks. It has been removed.